react-search/my-app/src/Items.tsx

```diff
@@ -0,0 +1,22 @@
+type Props = {
+  quotes: string[];
+};
+export const quotes = [
```

Comment:
This confused me. I was expect this to be in App.tsx instead of here. Ideally, you would have structured it so that in App.tsx you have:
```
const quotes = []......

<SearchableList quotes={quotes} />
```

You eventually get there, but structuring it as above would make it simpler.